salt/python_apps.sls

```diff
@@ -24,6 +24,16 @@ virtualenv:
 {% set directory = userdir + '/' + entry.git.target %}
 {% set context = {'name': name, 'entry': entry, 'appdir': directory} %}
 
+{% if entry.user == 'covid19admin'  %}
```

Comment:
During the provision, django app couldn’t get env, so i had to set manually in python app. Let me know what i missed during setup.

Reply:
Which env vars, and from where? Can you describe the issue more specifically? (e.g. error messages, etc.)

Reply:
used -env in cmd as you suggested, now the problem is resolved